packages/odyssey-react/src/components/Infobox/Infobox.theme.ts

```diff
@@ -19,8 +19,6 @@ export const theme: ThemeReducer = (theme) => ({
   BorderWidth: theme.BorderWidthBase,
   ColumnGap: theme.SpaceScale2,
   MarginBlockEnd: theme.SpaceScale3,
-  // eslint-disable-next-line @okta/odyssey/no-invalid-theme-properties
-  MaxLineLength: theme.FontLineLengthMax,
```

Comment:
this will be a breaking change we'll want to update the commit title to reflect the correct semantic version change `refactor!:` or similar, or look at another approach!